docs/pages/versions/unversioned/workflow/development-mode.md

```diff
@@ -1,26 +1,54 @@
 ---
-title: Development Mode
+title: Development and Production Mode
 old_permalink: /versions/v12.0.0/guides/development-mode.html
 previous___FILE: ./up-and-running.md
 next___FILE: ./configuration.md
 ---
 
-React Native includes some very useful tools for development: remote JavaScript debugging in Chrome, live reload, hot reloading, and an element inspector similar to the beloved inspector that you use in Chrome. It also performs bunch of validations while your app is running to give you warnings if you're using a deprecated property or if you forgot to pass a required property into a component, for example.
+import Video from '../../../../components/plugins/Video'
 
-![Screenshots of development mode in action](/static/images/development-mode.png)
+Your project will always run in one of two modes: development or production. By default, running your project locally with `expo start` runs it in development mode, whereas a published project (via `expo publish`), or any standalone apps, will run in production mode.
```

Comment:
currently the expo-cli behavior is to use the last mode that was used on the project. i can't remember where this was happening but some of us were discussing changing this so we don't persist this particular setting as it can be confusing for people, want to have a crack at opening a pr on expo-cli so we always start in dev mode?

Reply:
https://github.com/expo/expo-cli/pull/1162